schema/plotData.json

```diff
@@ -0,0 +1,79 @@
+{
+  "type": "object",
+  "description": "Daily plot data and metadata for a specific region ID and variable ID (found in filename by convention).",
+  "properties": {
+    "metadata": {
+      "properties": {
+        "minYear": {
+          "type": "integer",
+          "description": "The year in which the minimum for this region and variable occurred."
```

Comment:
I would say: The year in which the average of all days over the year (or median or sum, depending the variable, see supercomputer implementation) for this region and variable occurred.

Please improve my phrasing with better english if necessary

Reply:
> The year in which the average of all days over the year (or median or sum, depending the variable, see supercomputer implementation) for this region and variable occurred.

I'm a little confused about this phrasing, since it doesn't include the word "minimum". Maybe:

> The year in which the average (depending on the variable, could be median, mean, or sum) of all days in the year represents a climatological minimum for this region and variable.

?

Reply:
Maybe, based on your other commands, "climatological minimum" -> "historical minimum"?

Reply:
10c7bd4